src/commands/mfa.rs

```diff
@@ -32,8 +38,17 @@ fn request_flow(facility: &str) {
     super::wrap_in_spinner(|| client::mfa::submit(session, facility, &code), |r| r.message);
 }
 
-fn token_flow() {
-    let code = super::ask_user_input("Enter the code");
+fn token_flow(token: Option<&str>) {
+    let input_code: Option<String>;
+    let code: &str;
+
+    if let Some(t) = token {
+        code = t;
+    } else {
+        input_code = Some(super::ask_user_input("Enter the code"));
```

Comment:
The variable is only needed in this else block, so I would just drop the definition at the top and just use this:
```suggestion
        let input_code = Some(super::ask_user_input("Enter the code"));
```